TODO.md

```diff
@@ -1,9 +1,10 @@
-* continuous integration
```

Comment:
Is this actually doing anything? There's only changes in TODO. Also if we could do less refactoring in TODO that would help with reviewing :)

Reply:
This is still work in progress.

Yeah, sorry for the noise in TODO. Maybe it shouldn’t be checked in. Just organizing my thoughts and prioritizing.

Reply:
This is still work in progress.

Yeah, sorry for the noise in TODO. Maybe it shouldn’t be checked in. Just organizing my thoughts and prioritizing.

Reply:
Oh, I see. You opened a PR, so I thought this is ready. Was too quick :)

Reply:
Well you did as for my review, too, actually. Do you have the spec already?

Reply:
I think you were just added after you wrote a comment, but maybe I did accidentally add you a reviewer.